draft-ietf-quic-transport.md

```diff
@@ -2740,7 +2697,57 @@ that it sends.
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
-## Special Considerations for PMTU Discovery
+
+## Packet Size
+
+Clients MUST ensure that the first packet in a connection, and any
+retransmissions of those octets, has a QUIC packet size of least 1200 octets.
+The packet size for a QUIC packet includes the QUIC header and integrity check,
+but not the UDP or IP header.
+
+The initial client packet SHOULD be padded to exactly 1200 octets unless the
```

Comment:
s/initial client/QUIC initlal

Reply:
Perhaps "QUIC Initial and Handshake packets" ?

Reply:
It's only Initial.  I'll reword this.